### DISCUSSION TITLE
Feature suggestion: /withdraw

### DISCUSSION THREAD

**Discussion by scmcgowen** [EssentialsX/Essentials#4634] [Ideas and Feature Suggestions] [answered]
Add a /withdraw command to EssX.
Syntax: `/withdraw <amount>
This command will withdraw <amount> from the executing player's balance.
Every server that I've been on with an economy plugin has had /withdraw. I think it is a nice fit in EssX as it is one of the most basic things to be on a server.

**Comment by tadhgboyle** [comment]
What purpose does this serve?
If I execute `/withdraw 420`, I just... lose 420 balance?

**Reply by scmcgowen** [reply]
and get a bank note that you can store or give to another player

**Comment by JRoy** [comment, marked as answer]
This is out of the scope of the basic economy implementation Essentials has. A more specialized economy plugin should do this.
